bowling/tests/bowling_alley_tests_gt.cpp

```diff
@@ -0,0 +1,21 @@
+#include "gtest/gtest.h"
+
+#include "BowlingAlley.hpp"
+
+#include <filesystem>
+
+class BowlingAlleyTests : public ::testing::Test {
+protected:
+    std::filesystem::path path = "../games";
+    BowlingAlley bowlingAlley{path};
+
+};
+
+TEST_F(BowlingAlleyTests, bowlingAlleyCanBeDefaultCreated) {
+    BowlingAlley alley;
```

Comment:
disallow it. What does it mean that we have such BowlingAlley? It's useless to have such constructor.